package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a> # 📗 Table of Contents",
+  "main": "webpack.config.js",  
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
```

Comment:
- [ ] kindly consider adding a way to start your project to make your project accessible on the server hint: 
```
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
 
```